Cargo.toml

```diff
@@ -59,7 +57,9 @@ tracing-log = "0.1.0"
 tracing-tower = { git = "https://github.com/tokio-rs/tracing", rev = "65547d8809fcc726b8187db85b23c42e32ef5dce" }
 
 # Metrics
-hotmic = "0.8"
+metrics = "0.12.1"
+metrics-core = "0.5.2"
```

Comment:
```suggestion
metrics-core = "0.5"
```

Reply:
@Hoverbear What would `0.5` be better than `0.5.2`? `cargo` uses [caret requirements](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-cratesio) by default, so a newer patch release `0.5.3` would still be used instead of `0.5.2` if it becomes available.

Reply:
You're totally right!